packages/runtime/src/lib/constants.ts

```diff
@@ -307,22 +307,94 @@ AssetParamMap[6] = { ...AssetParamMap[5] };
 AssetParamMap[7] = { ...AssetParamMap[6] };
 AssetParamMap[8] = { ...AssetParamMap[7] };
 
+export enum AssetParamGetField {
```

Comment:
lets add links to some reference for all the enums

Reply:
Done